CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Master (Unreleased)
 
+## 1.19.0 (2017-10-18)
+
+Compatibility release so users can upgrade RuboCop. No new features
```

Comment:
Perhaps add the compatible version number here.

Reply:
True